Remove inaccurate content from research page

maya opened the pull request:
[😎 Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/remove-research-page)

Fixes #514.

hursey013 commented:
I think this page helps communicate that research is a core aspect of the USWDS and do think this page serves a purpose. I would agree that we should cut the User Research Activities
and Research as a Service Offerings section for now, but the introduction and section about analytics is still relevant in my opinion. 

Perhaps in a future PR the content of User Research Activities could be reintroduced to cover things like the utility class research that you are working on and other similar activities?

maya commented:
I added the Research page back with the following changes:
- Removed Research as a Service Offering and its references throughout the page
- Kept User Research Activities and Research, but removed Bi-annual Contextual Inquiries and Open House Events (with Remote Options) -- left in Digital Service Team Remote Interviews since we'll still do that
- Kept in Analytics Reporting but removed Dependencies b/c that seems more like a reminder to the USWDS team but not relevant to our users

hursey013 commented on pages/whats-new/research.md:
```diff
@@ -6,11 +6,10 @@ redirect_from:
-We plan to use a combination of research methods, quantitative research like collecting web analytics to see in how frequently the Design System is used around the federal government, and qualitative research like remote and in-person observational studies to see whether the Design System is making government sites easier for people to use.
 
-We’ll use this research to see how well the Design System is working and what needs to be improved. We can also provide user research services to teams that are using the Design System. These will be priced to allow teams around the federal government to engage with us and improve their product for their users. This benefits the Design System as well because we’ll get feedback on where they need  ...
+We plan to use a combination of research methods, quantitative research like collecting web analytics to see in how frequently the Design System is used around the federal government, and qualitative research like remote observational studies to see whether the Design System is making government sites easier for people to use. We’ll use this research to see how well the Design System is working a ...
```
Should the `in` be removed from `to see in how`?

maya replied:
Fixed in 7678f0c.

hursey013 commented on _data/nav.yml:
```diff
@@ -54,4 +54,3 @@ What’s new:
   - href: /whats-new/releases/
   - href: /whats-new/updates/
   - href: /whats-new/product-roadmap/
-  - href: /whats-new/research/
```
Need to add this back in

maya replied:
Fixed in 7678f0c.
